graph/repo.go

```diff
@@ -35,10 +35,10 @@ func TryMakeURI(cloneURL string) (string, error) {
 		return "", fmt.Errorf("MakeURI(%q): %s", cloneURL, err)
 	}
 
-	path := strings.TrimSuffix(url.Path, ".git")
-	path = filepath.Clean(path)
```

Comment:
This looks weird but since these are URLs not filepaths, using `filepath.Clean` was actually replacing `/`s in the URL with `\`s.